MovingBoxToy/test/circle.par

```diff
@@ -0,0 +1,59 @@
+ActiveThorns = "
+    MovingBoxToy
+    BoxInBox
+    CarpetX
+    CoordinatesX
+    IOUtil
+"
+
+Cactus::cctk_show_schedule = no
+Cactus::presync_mode = "mixed-error"
+
+CarpetX::poison_undefined_values = yes
+
+CarpetX::ncells_x = 64
+CarpetX::ncells_y = 64
+CarpetX::ncells_z = 64
+
+CarpetX::dtfac = CarpetX::ncells_x
+
+CarpetX::max_num_levels = 8
+CarpetX::regrid_every = 1
+
+BoxInBox::num_regions = 2
+
+BoxInBox::num_levels_1 = 2
+BoxInBox::position_x_1 = -0.5
+BoxInBox::radius_x_1[1] = 0.25
+BoxInBox::radius_y_1[1] = 0.25
+BoxInBox::radius_z_1[1] = 0.25
+
+BoxInBox::num_levels_2 = 2
+BoxInBox::position_x_2 = +0.5
+BoxInBox::radius_x_2[1] = 0.25
+BoxInBox::radius_y_2[1] = 0.25
+BoxInBox::radius_z_2[1] = 0.25
+
+CarpetX::boundary_x = "dirichlet"
+CarpetX::boundary_y = "dirichlet"
+CarpetX::boundary_z = "dirichlet"
+CarpetX::boundary_upper_x = "dirichlet"
+CarpetX::boundary_upper_y = "dirichlet"
+CarpetX::boundary_upper_z = "dirichlet"
+
+Cactus::cctk_itlast = 20
```

Comment:
20 iterations is a lot. Would 5 iterations suffice? That would make this test case 4 times smaller.